### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 _Keyboard: Tooltip for "split editor right" is not accessible with tab key and Arrow keys 

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17446]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Sep2022;#mssql for VSCode;#Win32;#DesktopApp;#A11ySev3;#Keyboard;#AINotInScope;#WCAG2.1.1;#Win11;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Visual studio code-insiders Version: 1.72.0-insider (user setup)

## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Open Visual studio code
3. Tab till extension button and search for "mssql"
4. Tab till install and move to the SQL Server button
5. Tab till Add Connection Button and insert the valid credentials
6. After adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
7. Select top 1000 window will open
8. Tab to the "split editor right" icon and check if the tooltip is available for the icon.

## Actual:
When using arrow keys and tab keys to navigate to "split editor right" icon, tooltip is not available with the keyboard keys.

**Note**: Similar issue is also observed in the following screens

1. Select top 1000 _Execute icon tooltip having shortcut key in it, Execute Query, More Actions buttons
2. Select top 1000 Results window_Icons Beside table in the results "Save as CSV, save as JSON, Save as excel"
3. Add connection button in the Database components section
4. Buttons in Query history

## Expected:
When using arrow keys and tab key to navigate to "split editor right" icon, tooltip should be available which is available with mouse, since shortcut keys were also mentioned in the tooltip.

## User Impact:
Keyboard users will not get the tooltip information.

## Attachment:
![17446_A11y_mssql for VSCode_Select top 1000 _Keyboard](https://user-images.githubusercontent.com/98821792/190948366-622fea2c-31e2-4a30-aa20-d136ff355243.gif)

**Comment by laurenastrid1** [comment]
Please use the latest version of the VS Code extension to verify and close this bug. This UI is no longer present in the extension.
